src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/configuration/KendraIntelligentRankingConfiguration.java

```diff
@@ -39,7 +39,7 @@ public class KendraIntelligentRankingConfiguration extends ResultTransformerConf
     PARSER = new ObjectParser<KendraIntelligentRankingConfiguration, Void>("kendra_intelligent_ranking_configuration", KendraIntelligentRankingConfiguration::new);
     PARSER.declareInt(TransformerConfiguration::setOrder, TRANSFORMER_ORDER);
     PARSER.declareObject(KendraIntelligentRankingConfiguration::setProperties,
-        (p, c) -> KendraIntelligentRankingProperties.parse(p, c),
+            KendraIntelligentRankingProperties::parse,
```

Comment:
There are several of refactor from method reference operator (::) in this PR. I am wondering if we should choose one direction to make the style more consistent

```
        new SearchExtSpec<>(SearchConfigurationExtBuilder.NAME, SearchConfigurationExtBuilder::new, SearchConfigurationExtBuilder::parse));
        new SearchExtSpec<>(SearchConfigurationExtBuilder.NAME,
                input -> new SearchConfigurationExtBuilder(input, resultTransformerMap),
                parser -> SearchConfigurationExtBuilder.parse(parser, resultTransformerMap)));
```

Reply:
IntelliJ recommends using the `::` operator where possible, so I usually autofix to that when it gets highlighted.

I'll double-check the rest of the PR to see if there are any that I missed.